libs/elektra/Makefile

```diff
@@ -0,0 +1,408 @@
+#
+# Copyright (C) 2006-2010 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+# No copyright by Harald Geyer as filling in a template doesn't constitute
+# an original work in the sense of copyright law.
+
+include $(TOPDIR)/rules.mk
+
+PKG_MAINTAINER:=Harald Geyer <harald@ccbib.org>
+
+PKG_NAME:=elektra
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=doc/COPYING
+PKG_VERSION:=0.8.15
+
+# Use this for official releasees
+PKG_MD5SUM:=33ec1e5982fb7fbd8893bf7b579b80f0
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://ftp.libelektra.org/ftp/elektra/releases
+
+# Use this to test versions still under development
+#PKG_SOURCE_PROTO:=git
+#PKG_SOURCE_URL:=https://github.com/ElektraInitiative/libelektra.git
+#PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+#PKG_SOURCE_VERSION:=e97efb29a94f3a49cb952d06552fcf53708ea8c7
+#PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.xz
+
+PKG_RELEASE:=1
+
+PKG_BUILD_DEPENDS:=elektra/host swig/host
+
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/libelektra-core
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Elektra core library
+  URL:=http://www.libelektra.org/
+  DEPENDS:=+libpthread
+endef
```

Comment:
I didn't use a default section because i think elektra-kdb belongs in utils, but I guess I can use the default section for anything but elektra-kdb. However one question: Your code above introduces a submenu "libelektra" - is there any policy about when to start a submenu?